samples/snippets/src/main/java/pubsublite/DeleteTopicExample.java

```diff
@@ -26,50 +25,35 @@
 import com.google.cloud.pubsublite.TopicName;
 import com.google.cloud.pubsublite.TopicPath;
 import com.google.cloud.pubsublite.TopicPaths;
-import io.grpc.StatusException;
 
 public class DeleteTopicExample {
 
   public static void main(String... args) throws Exception {
     // TODO(developer): Replace these variables before running the sample.
-    String CLOUD_REGION = "Your Cloud Region";
-    char ZONE_ID = 'b';
-    String TOPIC_NAME = "Your Topic Name";
-    long PROJECT_NUMBER = Long.parseLong("123456789");
+    String cloudRegion = "your-cloud-region";
+    char zoneId = 'b';
+    // Choose an existing topic.
+    String topicId = "your-topic-id";
+    long projectNumber = Long.parseLong("123456789");
 
-    DeleteTopicExample.deleteTopicExample(CLOUD_REGION, ZONE_ID, PROJECT_NUMBER, TOPIC_NAME);
+    deleteTopicExample(cloudRegion, zoneId, projectNumber, topicId);
   }
 
   public static void deleteTopicExample(
-      String CLOUD_REGION, char ZONE_ID, long PROJECT_NUMBER, String TOPIC_NAME) throws Exception {
-
-    try {
-      CloudRegion cloudRegion = CloudRegion.of(CLOUD_REGION);
-      CloudZone zone = CloudZone.of(cloudRegion, ZONE_ID);
-      ProjectNumber projectNum = ProjectNumber.of(PROJECT_NUMBER);
-      TopicName topicName = TopicName.of(TOPIC_NAME);
-
-      TopicPath topicPath =
-          TopicPaths.newBuilder()
-              .setZone(zone)
-              .setProjectNumber(projectNum)
-              .setTopicName(topicName)
-              .build();
-
-      AdminClientSettings adminClientSettings =
-          AdminClientSettings.newBuilder().setRegion(cloudRegion).build();
-
-      try (AdminClient adminClient = AdminClient.create(adminClientSettings)) {
-        adminClient.deleteTopic(topicPath).get();
-
-        System.out.println(topicPath.value() + " deleted successfully.");
-      }
-
-    } catch (StatusException statusException) {
-      System.out.println("Failed to delete the topic: " + statusException);
-      System.out.println(statusException.getStatus().getCode());
-      System.out.println(statusException.getStatus());
-      throw statusException;
+      String cloudRegion, char zoneId, long projectNumber, String topicId) throws Exception {
```

Comment:
All of these throw "Exception" - they should really be throwing a more specific Exception type, and catching and handling potential API errors in the code.

Reply:
The exception that needs handling from the autocloseable `try(AdminClient adminClient = AdminClient.create(adminClientSettings)` is `java.lang.Exception` in this case.

Reply:
If so, there should be a bug filed against the client. That seems like way too generic of an exception to be throwing from anything it should be doing. 

cc @lesv

Reply:
@lesv @kurtisvg See #148. Please continue the discussion there since this PR is merged.